src/main/java/se/iths/jakartaeelab1/service/PersonService.java

```diff
@@ -46,4 +46,6 @@ public void removePerson(UUID id){
         if (onePerson(id) != null)
             personRepository.deletePerson(id);
     }
+
+
```

Comment:
I only make this comment since I made other comments in the code, but remove these line breaks maybe so that they don't need to be in the PullRequest :)